runelite-client/src/main/java/net/runelite/client/plugins/zalcano/ZalcanoOverlay.java

```diff
@@ -169,6 +181,25 @@ private void renderRockToMine(Graphics2D graphics)
 		}
 	}
 
+	private void renderRockToMineAtMaP(Graphics2D graphics)
+	{
+		GameObject glowingRock = util.getGlowingRock();
+
+		if (glowingRock != null)
+		{
+			Point rockPoint = glowingRock.getMinimapLocation();
+			OverlayUtil.renderMinimapLocation(graphics, rockPoint, new Color(19, 5, 231));
+			Point rockPoint1 = new Point (rockPoint.getX() + 4, rockPoint.getY());
+			OverlayUtil.renderMinimapLocation(graphics, rockPoint1, new Color(249, 29, 151));
+			Point rockPoint2 = new Point (rockPoint.getX() - 4, rockPoint.getY());
+			OverlayUtil.renderMinimapLocation(graphics, rockPoint2, new Color(249, 29, 151));
+			Point rockPoint3 = new Point (rockPoint.getX(), rockPoint.getY() + 4);
+			OverlayUtil.renderMinimapLocation(graphics, rockPoint3, new Color(249, 29, 151));
+			Point rockPoint4 = new Point (rockPoint.getX(), rockPoint.getY() - 4);
+			OverlayUtil.renderMinimapLocation(graphics, rockPoint4, new Color(249, 29, 151));
```

Comment:
`Color` instances can be reused, and should be static final constants (if unconfigureable)

Can you add screenshots of added overlays?